week1/homework/src/index.js

```diff
@@ -1,3 +1,60 @@
 'use strict';
 
-// Write the homework code in this file
+const http = require('http');
+
+// state number
+let state = 10;
+
+// add and subtract function
+function add() {
+  state += 1;
+}
+
+function subtract() {
+  state -= 1;
+}
+// switch function to call add subtract ..
+function dataRequest(request, response) {
+  switch (request.url) {
+    case '/':
+      response.setHeader('Content-Type', 'text/html');
+      response.write(`<h1>${state}</h1>`);
+      break;
+    case '/state':
+      response.setHeader('Content-Type', 'text/html');
+      response.write(`<h1>${state}</h1>`);
+      break;
+    case '/add':
+      response.setHeader('Content-Type', 'text/html');
+      add();
+      response.write('<h1>OK</h1>');
+      response.write(`<h1>${state}</h1>`);
+      break;
+    case '/subtract':
+      response.setHeader('Content-Type', 'text/html');
+      subtract();
+      response.write('<h1>OK</h1>');
+      response.write(`<h1>${state}</h1>`);
+      break;
+    case '/reset':
+      response.setHeader('Content-Type', 'text/html');
+      const resetState = 10;
+      state = resetState;
+      response.write('<h1>OK</h1>');
+      response.write(`<h1>${state}</h1>`);
+      break;
+    // for bad url
+    default:
+      if (request.url !== '/' || 'state' || 'reset' || 'add' || 'subtract') {
```

Comment:
There is no need for this check since you are already excluding those cases by using the `switch` statement with those cases before.